index.html

```diff
@@ -7,140 +7,200 @@
+                <li><a href="#"><img src="images/links/medium.svg" alt="medium"></a></li>
+            </ul>
+        </div>
     </div>
 </section>
+
 <div class="works">
-    <h2>My Recent Works</h2>
-    <div class="sep"><hr></div>
+
+    <div class="works__title">
+        <h2 class="works__h2">My Recent Works</h2>
+        <div class="sep"><hr></div>
+    </div>
+
     <div class="multipost">
-        <img src="images/imgplaceholder.jpg" alt="Multi-Post">
-        <h3>Multi-Post Stories</h3>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-        <ul class="tags">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-        </ul>
-        <button type="button">See Project</button>
+        <div class="multipost__image">
+            <img src="images/imgplaceholder.jpg" alt="Multi-Post">
+        </div>
+        <div class="multipost__content">
+            <h3 class="multipost__h3">Multi-Post Stories</h3>
+            <p class="multipost__p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="multipost__tags">
+                <li class="multipost__tags__li">css</li>
+                <li class="multipost__tags__li">html</li>
+                <li class="multipost__tags__li">bootstrap</li>
+                <li class="multipost__tags__li">Ruby</li>
+            </ul>
+            <button type="button" class="multipost__btn">See Project</button>
+        </div>
     </div>
+
     <div class="cards">
-        <div class="card">
-            <div class="card_container">
-                <h3>Profesional Art Printing Data</h3>
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                <ul class="tags">
-                    <li>html</li>
-                    <li>bootstrap</li>
-                    <li>Ruby</li>
+
+        <div class="cards__card backcard01">
+            <div class="card__container">
+                <h3 class="card__h3">Profesional Art Printing Data</h3>
+                <p class="card__p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="card__tags">
+                    <li class="card__tags__li">html</li>
+                    <li class="card__tags__li">bootstrap</li>
+                    <li class="card__tags__li">Ruby</li>
                 </ul>
             </div>
-            <button  type="button">See Project</button>
+            <button type="button" class="card__btn">See Project</button>
```

Comment:
- [ ] Kindly edit the desktop version of the card to make it match Figma; when the project is hovered on with the mouse the description and dark opacity disappears and a `See Project` button appears at the bottom of the image and takes the whole image width. 

## Default state:
![image](https://user-images.githubusercontent.com/50721479/186996458-b47772b5-31bf-4047-8695-c4eebe400d85.png)

## On hover:
![image](https://user-images.githubusercontent.com/50721479/186996487-bd4e5016-58f2-4182-a7d0-4afcbd32baae.png)

Reply:
I have tried several ways but I have not been able to make the entire section hide and the button appear with a single hover, I have searched a lot but I have not found a solution that corrects the problem. 😢 

I would be very grateful if you could give me more exact information about it since it is the only thing I need to finish everything. Thank you very much.